AdminMessage/CronMessage.php

```diff
@@ -89,7 +92,8 @@ public function isDisplayed()
      */
     public function getText()
     {
-        $message = __('You have ' . $this->_cronCheck . ' unprocessed notification(s). Please check your Cron');
+        $urlNotificationsOverview=$this->backendHelper->getUrl("adyen/NotificationsOverview/index");
```

Comment:
please check your formatting settings in your code editor.

Reply:
PSR2 check done